scripts/qa_crowbarsetup.sh

```diff
@@ -3338,12 +3338,14 @@ function oncontroller_run_tempest
 
     if iscloudver 6plus; then
         tempest cleanup --init-saved-state
-        local opts=$tempestoptions
+        local opts=($tempestoptions)
         if iscloudver 7plus; then
             ### backward compatibility, remove
-            opts=${opts/-s/--smoke}
-            opts=${opts/-t/--serial}
-            tempest run $opts 2>&1 | tee tempest.log
+            for i in ${!opts[@]}; do
+                opts[$i]=${opts[$i]/-s/--smoke}
```

Comment:
this still has the same problem as before, you can match on start of string with an extra '#'
```bash
opts[$i]=${opts[$i]/#-s/--smoke}
```